portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsCreation.testcase

```diff
@@ -1043,7 +1077,7 @@ definition {
 			ACSegments.viewSegmentCriteriaIndividualAttributes(
 				attributeName = "age",
 				attributeValue = "40",
-				condition = "less than");
+				condition = "greater than");
 		}
 
 		task ("Switch tab to membership and check the membership list") {
```

Comment:
Hey @manuferreira, I think at the end of the test you could put it to verify that user2 is not in the list, that way we can verify that the criteria are working in the segment:

`ACUtils.viewNameListIsNotPresent(nameList = "user2 user2");`